base/Terminals.jl

```diff
@@ -101,8 +101,8 @@ end
 mutable struct TTYTerminal <: UnixTerminal
     term_type::String
     in_stream::Base.TTY
```

Comment:
I thought TTYTerminal was only intended to be used with TTY input streams. `_start` checks for that at least.